interfaces/builtin/desktop_test.go

```diff
@@ -92,6 +92,15 @@ func (s *DesktopInterfaceSuite) TestSanitizePlug(c *C) {
 }
 
 func (s *DesktopInterfaceSuite) TestAppArmorSpec(c *C) {
+	tmpdir := c.MkDir()
+	dirs.SetRootDir(tmpdir)
+	c.Assert(os.MkdirAll(filepath.Join(tmpdir, "/usr/share/fonts"), 0777), IsNil)
+	c.Assert(os.MkdirAll(filepath.Join(tmpdir, "/usr/local/share/fonts"), 0777), IsNil)
+	c.Assert(os.MkdirAll(filepath.Join(tmpdir, "/var/cache/fontconfig"), 0777), IsNil)
+
+	restore := release.MockOnClassic(false)
```

Comment:
So this would pretend to be on a core device? Is that what you intended?

Reply:
It looks like I copied a few too many lines from TestMountSpec.  Removed.